### PR TITLE
Pull in translations DAH-1064

### PR DESCRIPTION
**Pull request by akegan**
DAH-1064

Pulls in all the latest translations. Review instructions in the ticket. 

I also added "notranslate" tags to some habitat content to prevent any potential issues with google translate trying to retranslate that content.

I also added notranslate to these preference numbers while I was at it because this looked really wacky
image

### CODE REVIEW

File: app/assets/json/translations/locale-es.json [hunk 1]
```diff
@@ -1043,6 +1097,14 @@
                 "step_5": "Tenga ahorros suficientes para los costos de cierre y el pago inicial",
                 "title": "Antes de presentar la solicitud"
             },
+            "before_applying_habitat": {
+                "read_full_list": "Para obtener más detalles, lea <a href='{{url}}' target='_blank'>la lista completa de requisitos</a>.",
+                "rules_are_different": "Estas unidades poseen normas diferentes de otros anuncios de propiedad en DAHLIA. Asegúrese de:",
+                "step_1": "asistir a una <a href='{{url}}' target='_blank'>sesión informativa de Habitat for Humanity</a>",
+                "step_2": "no haber sido propietario de una propiedad residencial en los últimos 3 años",
+                "step_3": "cumplir con los <a href='{{url}}' target='_blank'>requisitos de ingresos</a>",
+                "step_4": "tener una puntuación de crédito de 650 o mayor"
```
Comment (christine-sfg) [inline app/assets/json/translations/locale-es.json:1106]: Should the first words for each of these steps be capitalized?

File: app/assets/json/translations/locale-es.json [hunk 2]
```diff
@@ -903,6 +924,39 @@
             "final_waitlist_size": "Tamaño de la lista de espera final",
             "for_household_size": "para <strong>{{size}}</strong> {{people}}",
             "for_income_calculations": "Para el cálculo de los ingresos, tenga en cuenta que el tamaño del grupo familiar incluye a todas las personas (de cualquier edad) que viven en la unidad.",
+            "habitat": {
+                "application_process": {
+                    "ol1": "Debe asistir a una <a target='_blank' href='{{infoSessionLink}}'>sesión informativa de Habitat for Humanity</a>.",
+                    "ol10": "¡Puede mudarse a su nueva casa!",
+                    "ol2": "Presentará su solicitud en DAHLIA.",
+                    "ol3": "La MOHCD realizará el sorteo y publicará los resultados aquí en DAHLIA.",
```
[the remark this answers is not in the document]
Comment (akegan) [reply app/assets/json/translations/locale-es.json:932]: I think I'm going to leave this as is. It'd be nice to have a spanish bilingual content strategist on the team, but I don't think we have the expertise as of right now to decide what makes most sense here. We have "la MOHCD" elsewhere in the doc.

File: app/assets/json/translations/locale-es.json (hunk 1 above)
Comment (akegan) [reply app/assets/json/translations/locale-es.json]: just pushed this change

File: public/ie-deprecated.html [hunk 3]
```diff
@@ -35,23 +35,23 @@
...
 
     .message__title {
       font-size: 24px;
       font-weight: 100;
-      margin-top: 15px;
+      margin-top: 16px;
       color: #47494E;
-      margin-bottom: 25px;
-      max-width: 800px;
+      margin-bottom: 16px;
```
Comment (akegan) [inline public/ie-deprecated.html:52]: I paired with Yindi Friday afternoon to tweak the styling

**Review by christine-sfg — APPROVED** [review]

**Pull request merged**